distage/distage-core-api/src/main/scala/izumi/distage/model/Injector.scala

```diff
@@ -122,10 +122,19 @@ trait Injector extends Planner with Producer {
   final def produceGetF[F[_]: TagK: DIEffect, A: Tag](bindings: ModuleBase, activation: Activation): DIResourceBase[F, A] = {
     produceF[F](plan(PlannerInput(bindings, activation, DIKey.get[A]))).map(_.get[A])
   }
+
+  final def produceGetF[F[_]: TagK: DIEffect, A: Tag](bindings: ModuleBase): DIResourceBase[F, A] = {
+    produceGetF[F, A](bindings, Activation.empty)
+  }
+
   final def produceGetF[F[_]: TagK: DIEffect, A: Tag](name: Identifier, activation: Activation)(bindings: ModuleBase): DIResourceBase[F, A] = {
     produceF[F](plan(PlannerInput(bindings, activation, DIKey.get[A].named(name)))).map(_.get[A](name))
   }
 
+  final def produceGetF[F[_]: TagK: DIEffect, A: Tag](name: Identifier)(bindings: ModuleBase): DIResourceBase[F, A] = {
```

Comment:
@coreyoconnor I've pushed a fix for this here: https://github.com/7mind/izumi/commit/d388a1d287af6a7e2a80051108c13a8f686228f3#diff-73492634fa478a23404fb7002c2dfe65R125 so you may remove it since there's now a merge conflict